src/components/txNotification/index.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2018-present, Parsec Labs (parseclabs.org)
+ *
+ * This source code is licensed under the GNU GENERAL PUBLIC LICENSE Version 3
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+import 'antd/dist/antd.css';
+
+import React from 'react';
+import { observer, inject } from 'mobx-react';
+import PropTypes from 'prop-types';
+import { notification } from 'antd';
+import { TxStatus } from './types';
+
+
+const statusDetails = {};
+statusDetails[TxStatus.CREATED] = { text: 'Waiting for signature..', icon: 'warning' };
+statusDetails[TxStatus.INFLIGHT] = { text: 'Mining..', icon: 'info' };
+statusDetails[TxStatus.SUCCEED] = { text: 'Success', icon: 'success' };
+statusDetails[TxStatus.FAILED] = { text: 'Transaction failed', icon: 'error' };
+statusDetails[TxStatus.CANCELLED] = { text: 'Cancelled', icon: 'error' };
+
```

Comment:
```es6
const statusDetails = {
  [TxStatus.CREATED]: { text: 'Waiting for signature..', icon: 'warning' },
  [TxStatus.INFLIGHT]: { text: 'Mining..', icon: 'info' },
  [TxStatus.SUCCEED]: { text: 'Success', icon: 'success' },
  [TxStatus.FAILED]: { text: 'Transaction failed', icon: 'error' },
  [TxStatus.CANCELLED]: { text: 'Cancelled', icon: 'error' },
};
```

:-)

Reply:
Cool! Thanks. What kind of syntax is this?

Reply:
Computed property keys from ES6. You can put any expression in square brackets:

```
console.log({ [Math.random()]: 'random key' });
```